### PR TITLE
feat: add transfer recovery from lock/withdraw hash

### PR DESCRIPTION
**Pull request by paouvrard**
Implementation for https://github.com/near/rainbow-bridge-frontend/issues/136
Requires client release with https://github.com/near/rainbow-bridge-client/pull/10
Given an eth lock hash or a near withdraw hash, recreate a transfer object which can be finalized (mint/unlock)

### CODE REVIEW

File: src/html/transfers.html [hunk 1]
```diff
@@ -7,9 +7,38 @@ <h1>Your Transfers</h1>
     </button>
     <!-- TODO: <nav> -->
   </header>
+  <div class="recovery-container">
+    <button class="link" data-behavior="toggleRecovery">Recover with deposit tx hash</button>
```
Comment (chadoh) [inline src/html/transfers.html:11]: A design question: "deposit tx hash" does not indicate "Ethereum lock transaction hash OR NEAR withdraw transaction hash" to me. Not sure what it should say instead. Can wait to fix this later though.

**Review by chadoh — APPROVED** [review]

File: src/html/transfers.html (hunk 1 above)
Comment (paouvrard) [reply src/html/transfers.html:11]: This will be improved with the new design. But the tx hash input field can accept both an Ethereum or Near tx hash.

**Comment by paouvrard** [comment]
image
image
image
image
image

**Pull request merged**